.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Stale issue handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@main
```

Comment:
This should use a tag such as `@v4` instead of `@main` so that it won't break when newer version are released.

Reply:
👍 Was going to change after I copied, then I forgot about it. 😬. Will address in a follow-up PR.